### PR TITLE
Improve template

### PR DESCRIPTION
**Pull request by Danie**
Hello,

I created two changes regarding the template:

1. change easily date, day, time format in main input section
2. dynamic vehicle type label so that only available vehicles shown

Regards,

Daniel

### CODE REVIEW

**Comment by willgeary** [comment]
This looks amazing! Will review, thank you @Danie !

**Comment by willgeary** [comment]
I love this @Danie .  However it can be a bit distracting and hard to follow at times when there are many quick changes to the labels. I think I will include this as a second template option in `templates/` called `dynamic_labels.pde` so that users can request dynamic labels with a`--dynamic_labels` command line argument. Thoughts?

**Comment by Danie** [comment]
Hello @willgeary - I know what you mean. It could be very "nervous" to watch on it. I changed some more things and now only vehicles are showing that are now or were in the past in flow.

But your idea with the second template is also an option.

**Comment by willgeary** [comment]
Very cool @Danie , thank you! Will review it.

**Comment by willgeary** [comment]
Merged, thanks @Danie ! It's a great improvement
